packages/astro/src/transitions/router.ts

```diff
@@ -515,7 +515,7 @@ if (inBrowser) {
 		addEventListener('popstate', onPopState);
 		addEventListener('load', onPageLoad);
 		if ('onscrollend' in window) addEventListener('scrollend', onScroll);
-		else addEventListener('scroll', throttle(onScroll, 300));
+		else addEventListener('scroll', throttle(onScroll, 350));
```

Comment:
Might as well make this a passive event listener at the same time, right? [It's widely supported](https://caniuse.com/passive-event-listener) these days.

```suggestion
		else addEventListener('scroll', throttle(onScroll, 350), { passive: true });
```